### ISSUE TITLE
River allows keyboard focus on layer shell applications when a view is in fullscreen

### ISSUE THREAD

**Issue by zakvf** [riverwm/river#559]
Note: My river version is 0.2.0-dev-422cd3e.

When a view is in fullscreen mode, river allows keyboard focus to be taken away when spawning a layer shell application requiring keyboard interactivity. The new layer shell application is not visible, however, so it is not obvious where keyboard input goes.

Steps to reproduce:

1.  Place a view in fullscreen mode.
2.  Run a layer shell application requiring keyboard input (in my case, fuzzel).

The above should result in keyboard focus being taken away from the fullscreen view with no way to return the focus until the layer shell application is closed.

I encountered this issue when I had a fullscreen terminal (foot) open and accidentally pressed my keybinding to open fuzzel. I then realised that my keyboard focus was on an invisible application, which was counterintuitive behaviour for me. Is there any way to show layer shell applications with keyboard focus above all other applications (including fullscreen ones)?

**Comment by zakvf** [comment]
Just found out that this is not an issue with river after looking through the code and doing some more research (see https://codeberg.org/dnkl/fuzzel/issues/81). Apologies for the confusion. Closing
